package/setup.py

```diff
@@ -83,7 +84,6 @@
     import Cython
     from Cython.Build import cythonize
     cython_found = True
-    from packaging.version import Version
```

Comment:
So thinking about it a bit more, this explicitly makes `packaging` a core requirement for user builds right?

I guess the question is - where do we stand on eventually making Cython a core dependency (rather than us shipping the C/C++ files)? If we want to keep things the way we usually do, then I think we should possibly try/except around the packaging import and just set it to either say that either cython or packaging isn't found?

Reply:
> So thinking about it a bit more, this explicitly makes packaging a core requirement for user builds right?

What? I just moved the import so the error message would be better when `packaging` is absent--it was the MDA team who added this dependency months ago.